ampbench_lib.js

```diff
@@ -775,10 +775,14 @@ function lib_fetch_cdn_validator_spec_file_revision(callback) {
  */
 function lib_load_validator(opt_force_reload) {
     let _force_reload = opt_force_reload || false;
-    const _validator = amphtml_validator.getInstanceFromFileSync(VALIDATOR_JS_FILE, _force_reload);
-    amphtml_validator_instance = _validator; // cache the module level instance here
-    amphtml_validator_spec_file_revision = lib_extract_validator_spec_file_revision(_validator.scriptContents);
```

Comment:
Agreed. For the interim I have replaced the validator spec file revision with a signature based on simple validator code size as per: https://github.com/ampproject/ampbench/commit/040a3bf8413520368bf3d5385dcf31e0f8f09849